packages/thirdweb/src/pay/swap/actions/getSwap.ts

```diff
@@ -0,0 +1,165 @@
+import { getClientFetch } from "../../../utils/fetch.js";
+import { type ThirdwebClient } from "../../../client/client.js";
+import { THIRDWEB_PAY_SWAP_ROUTE_ENDPOINT } from "../utils/definitions.js";
+
+export type SwapRouteParams = {
+  client: ThirdwebClient;
+  fromAddress: string;
+  fromChainId: number;
+  fromTokenAddress: string;
+  fromAmountWei?: string;
+  toAddress?: string;
+  toChainId: number;
+  toTokenAddress: string;
+  toAmountWei?: string;
+  maxSlippageBPS?: number;
+};
+
+type Approval = {
+  chainId: number;
+  tokenAddress: string;
+  spenderAddress: string;
+  amountWei: string;
+};
```

Comment:
this looks suspiciously like the input params for the `approve()` extension, can we align them and use the same one?